main/src/cgeo/geocaching/network/StatusUpdater.java

```diff
@@ -64,6 +66,22 @@ static Status defaultStatus(final Status upToDate) {
         }
     }
 
+    private static String getActiveConnectorsString() {
+        final IConnector[] activeConnectors = ConnectorFactory.getActiveConnectors();
+        final StringBuilder activeConnectorsBuilder = new StringBuilder();
+        for (final IConnector conn : activeConnectors) {
+            try {
+                activeConnectorsBuilder.append(conn.getNameAbbreviated());
+                activeConnectorsBuilder.append(',');
+            } catch (IllegalStateException ignored) {
+            }
+        }
+        if (activeConnectorsBuilder.length() > 0) {
+            activeConnectorsBuilder.deleteCharAt(activeConnectorsBuilder.length() - 1);
+        }
+        return activeConnectorsBuilder.toString();
```

Comment:
This looks overly complicated to me. Any way to use `StringUtils.join()` or similar instead of reinventing the wheel to join strings with a separator? Even constructing an intermediate list would be more readable IMO, and this method will be called once every 30 minutes as far as cgeo-status goes.

Reply:
The problem is that we need to call extra method for every element in collection, so you cannot apply `map` so easily. If you think that intermediate list will be better (I thought exactly opposite and that's why I used StringBuilder), I can replace